api/build.gradle

```diff
@@ -724,6 +725,13 @@ task listProjectAPIs(type: GenerateAPIListingTask)
 def db_host = System.getenv("DB_HOST")
 def db_port = System.getenv("DB_PORT")
 
+// run with "./gradlew reportLibyears"
+libyear {
+    configurations = ['compileClasspath']
+    failOnError = false // necessary for sam-client
+    validator = singleArtifactMustNotBeOlderThan(years(10))
```

Comment:
we're actually pushing this one!

Reply:
In that case, maybe this validation should be removed or significantly increased. I wouldn't want someone to try the tool and have it error out, which can be demotivating. At the moment, I'd consider libyears a tool for _learning_ about our dependency freshness rather than _enforcing_ it.